pkg/irreparable/irreparable.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/migrate"
+	dbx "storj.io/storj/pkg/irreparable/dbx"
+)
+
+var (
+	mon = monkit.Package()
+)
+
+// Database implements the irreparable RPC service
+type Database struct {
+	mu     sync.Mutex
+	DB     *dbx.DB
+	logger *zap.Logger
```

Comment:
Remove the logger, database shouldn't really log... it rarely shows something useful.

Reply:
done